### PR TITLE
Fixed some issues

### PR DESCRIPTION
**Pull request by alaarman**
As there is a workaround for the NDFS issue, I won't focus on it.
@yanntm request for a fix for #145 has been addressed though, as well as a fix for #135.

### CODE REVIEW

**Comment by yanntm** [comment]
Awesome, Thanks a lot Alfons.

**Comment by alaarman** [comment]
Fixed in d5660bb6c4237f74038acfd6f8d6afdd5b4ab629

**Comment by alaarman** [comment]
Should I create a new pull request @Meijuh ?

**Comment by Meijuh** [comment]
@alaarman, why? This pull request is fine right?

**Comment by alaarman** [comment]
I didn't know the pull request would be updated automatically. Take this one :)

File: src/mc-lib/cctables.c [hunk 1]
```diff
@@ -101,9 +101,11 @@ put_chunk (value_table_t vt,chunk item)
 static void
 put_at_chunk (value_table_t vt,chunk item,value_index_t pos)
 {
```
Comment (alaarman) [inline src/mc-lib/cctables.c]: @bergfi this fixes the issues we discussed.
Alternatively, the read/write lock can be used in put/get as well, but that would slow down chunk handling beyond anything acceptable.

**Comment by alaarman** [comment]
@Meijuh 
The tests on Linux fail because Sylvan is missing?

**Pull request merged**